### ISSUE TITLE
loss returns a very value, when I apply it to my binary segment task using U-net

### ISSUE THREAD

**Issue by bubusang** [Atomwh/FocalLoss_Keras#4]
Thanks for your work firstly! 

Now my problem is that when I apply this focal loss to my segmentation task, the loss return value is very large! It can go to 20000 or even larger! 
(It is over 1000 even when the network is well-trained.) 

> 844/844 [==============================] - 71s 84ms/step - loss: 32905.6789 - dice_coef: 0.3220 - acc: 0.8944 - val_loss: 21817.5009 - val_dice_coef: 0.3578 - val_acc: 0.9168
Epoch 2/100
844/844 [==============================] - 58s 68ms/step - loss: 20204.5861 - dice_coef: 0.3671 - acc: 0.9097 - val_loss: 19011.6854 - val_dice_coef: 0.3672 - val_acc: 0.9208

When I change the loss function into 'binary_cross_entropy', the loss value is normal, (between 0 and 1.0). 

I want to know that does such strange result occur in your work. 

In addition, unlike your cell segment task, my segment targets is small, and I labeled them in white. 

Thank you in advance!
